code/modules/antagonists/eldritch_cult/knowledge/rust_lore.dm

```diff
@@ -0,0 +1,198 @@
+		if(E)
+			E.on_effect()
+			H.adjustOrganLoss(pick(ORGAN_SLOT_BRAIN,ORGAN_SLOT_EARS,ORGAN_SLOT_EYES,ORGAN_SLOT_LIVER,ORGAN_SLOT_LUNGS,ORGAN_SLOT_STOMACH,ORGAN_SLOT_HEART),25)
+	target.rust_heretic_act()
+	return TRUE
+
+/datum/eldritch_knowledge/spell/area_conversion
+	name = "Agressive Spread"
+	desc = "Spreads rust to nearby turfs. Destroys already rusted walls."
+	gain_text = "All men wise know not to touch the bound king."
+	cost = 1
+	spell_to_add = /obj/effect/proc_holder/spell/aoe_turf/rust_conversion
+	next_knowledge = list(/datum/eldritch_knowledge/rust_blade_upgrade,/datum/eldritch_knowledge/curse/corrosion,/datum/eldritch_knowledge/spell/blood_siphon)
+	route = PATH_RUST
+
+/datum/eldritch_knowledge/rust_regen
+	name = "Leeching Walk"
+	desc = "Passively heals you when you are on rusted tiles."
+	gain_text = "The strength was unparallel, it was unnatural. Blacksmith was smiling."
+	cost = 1
+	next_knowledge = list(/datum/eldritch_knowledge/rust_mark,/datum/eldritch_knowledge/armor,/datum/eldritch_knowledge/essence)
+	route = PATH_RUST
+
+/datum/eldritch_knowledge/rust_regen/on_life(mob/user)
+	. = ..()
+	var/turf/user_loc_turf = get_turf(user)
+	if(!istype(user_loc_turf, /turf/open/floor/plating/rust) || !isliving(user))
+		return
+	var/mob/living/living_user = user
+	living_user.adjustBruteLoss(-2, FALSE)
+	living_user.adjustFireLoss(-2, FALSE)
+	living_user.adjustToxLoss(-2, FALSE)
+	living_user.adjustOxyLoss(-0.5, FALSE)
+	living_user.adjustStaminaLoss(-2)
+
+/datum/eldritch_knowledge/rust_mark
+	name = "Mark of Rust"
+	desc = "Your eldritch blade now applies a rust mark. Rust mark has a chance to deal between 0 to 200 damage to 75% of enemies items. To Detonate the mark use your mansus grasp on it."
+	gain_text = "Lords of the depths help those in dire need at a cost."
+	cost = 2
+	next_knowledge = list(/datum/eldritch_knowledge/spell/area_conversion)
+	banned_knowledge = list(/datum/eldritch_knowledge/ash_mark,/datum/eldritch_knowledge/flesh_mark)
+	route = PATH_RUST
+
+/datum/eldritch_knowledge/rust_mark/on_eldritch_blade(target,user,proximity_flag,click_parameters)
+	. = ..()
+	if(isliving(target))
+		var/mob/living/living_target = target
+		living_target.apply_status_effect(/datum/status_effect/eldritch/rust)
+
+/datum/eldritch_knowledge/rust_blade_upgrade
+	name = "Toxic blade"
+	gain_text = "Let the blade guide you through the flesh."
+	desc = "Your blade of choice will now add toxin to enemies bloodstream."
+	cost = 2
+	next_knowledge = list(/datum/eldritch_knowledge/spell/rust_wave)
+	banned_knowledge = list(/datum/eldritch_knowledge/ash_blade_upgrade,/datum/eldritch_knowledge/flesh_blade_upgrade)
+	route = PATH_RUST
+
+/datum/eldritch_knowledge/rust_blade_upgrade/on_eldritch_blade(target,user,proximity_flag,click_parameters)
```

Comment:
Heretic has very few offensive abilities, toxic blade is probably the best. This is strong, but it is weaker than things like shotguns or stun batons the crew has access to at round start.

Reply:
i mean with hitting yourself with the blade you are trading around 17 damage now for 30 healing over time, i dont think that is too op. Ive been observing heretics on tg for quite some time and lemme tell you, this doesnt offset how hard the antag overall is.

Reply:
> i mean with hitting yourself with the blade you are trading around 17 damage now for 30 healing over time, i dont think that is too op. Ive been observing heretics on tg for quite some time and lemme tell you, this doesnt offset how hard the antag overall is.

the hitting yourself is not something i'm worried about at all, though rust gets enough armor to offset that damage (which is only 10-20 healing), but i'm more worried about how every strike nets 47 damage over time, making 3 strikes on unarmored opponents lethal, and 4 on armored opponents.

Reply:
well it is fairly late into the skill tree, it would take around 20 minutes of getting influences and like 5 sacrifices to get to that point. I guess i can decrease that to 2-3 u.

Reply:
do that and i'll see about a testmerge.